src/tr.py

```diff
@@ -1,22 +1,37 @@
+"""
+Translating text
```

Comment:
Is this docstring okay? @g1itch  @PeterSurda

Reply:
Some day it will become so: https://github.com/g1itch/PyBitmessage/blob/qt5-wip/src/tr.py
But now I'm not sure which text is more descriptive.